+Base/GitPanel.m

```diff
@@ -0,0 +1,164 @@
+classdef GitPanel
+    
```

Comment:
Can you add a small comment section describing this class?

Reply:
See latest commit.